src/components/RadioButton/style.scss

```diff
@@ -51,7 +52,7 @@ $base-class: 'radio';
     font-size: 15px;
     color: $font-primary-color;
     line-height: 20px;
-    flex: 1;
+    flex: 1 0 auto;
```

Comment:
can you fix `.lc-checkbox__text` as well? there is the same problem

Reply:
and `.lc-checkbox__label`

Reply:
@kamilmateusiak done